### PR TITLE
5-getJavaScriptForWebView-extra-param: updating Android tests

### PR DESCRIPTION
**Pull request by felipe-saez**
This PR contains:
- Updates to the Android UI tests in order to work with the new testing app.

### CODE REVIEW

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/ScrollToAction.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        return "scroll to"
+    }
+
+    companion object {
+        private val TAG = ScrollToAction::class.java.simpleName
+        fun betterScrollTo(): ViewAction {
+            return ViewActions.actionWithAssertions(ScrollToAction())
+        }
+    }
+}
```
Comment (nicolas-chaix-didomi) [inline testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/ScrollToAction.kt:61]: please add a blank line at the end of the file

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt [hunk 2]
```diff
@@ -36,51 +35,95 @@ class UIGettersTest {
     fun tearDown() {
```
Comment (nicolas-chaix-didomi) [inline testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt:35]: you can remove this function

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt [hunk 3]
```diff
@@ -36,51 +35,95 @@ class UIGettersTest {
...
-        testMethodCall("getEnabledVendorIds", true)
+        agreeToAll()
+
+        tapButton("getEnabledVendorIds".toUpperCase())
+        assertText(ALL_VENDOR_IDS)
+    }
+
+    private fun agreeToAll() {
+        tapButton("setUserAgreeToAll".toUpperCase())
+        Thread.sleep(2000L)
```
Comment (nicolas-chaix-didomi) [inline testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt:104]: In Java / Kotlin you can insert underscore to make a number more readable.
2000L can be 2_000L
2000000 can be 2_000_000

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt [hunk 4]
```diff
@@ -36,51 +35,95 @@ class UIGettersTest {
...
+    private fun assertText(text: String) {
+        val matcher = withText(text)
+        onView(matcher).perform(ScrollToAction())
+    }
+
+    companion object {
+        const val ALL_VENDOR_IDS = "28,google"
+        const val ALL_PURPOSE_IDS = "cookies,create_ads_profile,geolocation_data,select_personalized_ads"
     }
 }
```
Comment (nicolas-chaix-didomi) [inline testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt:129]: missing blank line at the end of the file

**Review by pmerlet-at-didomi — APPROVED** [review]

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt (hunk 2 above)
Comment (felipe-saez) [reply testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt]: done

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt (hunk 3 above)
Comment (felipe-saez) [reply testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt]: done

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt (hunk 4 above)
Comment (felipe-saez) [reply testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/UIGettersTest.kt]: done

File: testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/ScrollToAction.kt (hunk 1 above)
Comment (felipe-saez) [reply testApp/android/app/src/androidTest/java/com/example/reactnativedidomi/ScrollToAction.kt]: I ended up removing this file

**Pull request merged**